server/models.js

```diff
@@ -29,3 +29,13 @@ exports.fetchArticleById = (id) => {
       return data.rows[0];
     });
 };
+
+exports.fetchArticles = () => {
+  return db
+    .query(
+      "SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;"
```

Comment:
Just be aware that your comment count will come back as a string and not a digit! This is fine, but just wanted to make you aware.